web/locales/en/messages.ftl

```diff
@@ -322,7 +322,7 @@ skip-submission-note = Note: You will still need to select between Speak or List
 off = Off
 on = On
 add-avatar-title = Add an avatar to your profile
-add-avatar-clip = Add an avatar to your profile and say “hello!”
+add-avatar-clip = Give your avatar a voice
```

Comment:
can we change the id of this string? that way communities in other languages will be prompted to add a new string

Reply:
Yeah, right